### PR TITLE
enabled nano cache size

### LINKED ISSUE #1835: Introduce cache_size Nano
OC.NL has re-introduced the size Micro again.
To go a step further than only show micro (like GC) it should be nice to also show Nano caches in the system.

As suggestion the Nano could have ID 1 because the Micro has ID 2. The bigger the caches sizes, the biggest the ID number.

Because the search page looks at the normal DB this can easy be solved by importing the table "cache_size" that can be uploaded in /sqlAlters. The search page would loo like:
selectie_289

Is it possible to add this Nano size as ID 1 to the system?

### PR DESCRIPTION
**Pull request by following5**
This fixes #1835.

* Cache size "Nano" is enabled for all OC sites. If some site does not want it, it can be disabled in Config/site.XX.php.
* Cache size "Not specified" is enabled only for OC RO, where some of those caches exist. https://github.com/opencaching/okapi/issues/519 should be fixed asap.
* All size search and filter pages adjust to the existing caches. E.g. "Nano" search option will appear when the first Nano cache is published on the site.
* Table `cache_size` becomes obsolete.
* Fixed some PHP notices on 'mobile' pages.

### CODE REVIEW

**Comment by deg-pl** [comment]
Please disable nano size by default on all OC sites, and enable only for OC NL.

File: lib/Objects/GeoCache/GeoCache.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
 namespace lib\Objects\GeoCache;
 
+use Utils\Database\OcDb;
```
Comment (deg-pl) [inline lib/Objects/GeoCache/GeoCache.php]: @following5 You have acces to OcDb via $this->db (from BaseObject). So don't use OcDb this way please.

Comment (following5) [reply lib/Objects/GeoCache/GeoCache.php]: I could use explicit `\lib\Objects\OcConfig\OcDb::instance()` to get rid of the use statement. But there are lots of other static GeoCache:: methods that currently use XDb and will reference OcDb class when refactored to OcDb.

Comment (deg-pl) [reply lib/Objects/GeoCache/GeoCache.php]: Possible in static method via self::db() (see BaseObject)

**Comment by following5** [comment]
This change also fixes issues #600 and #876.

**Pull request merged**